ck3-tiger.conf

```diff
@@ -65,25 +110,30 @@ output_style = {
     # Also, all other settings in this block will be ignored.
     enable = yes
 
+    # The maximum line-length of the output.
+    # Set this to 0 to disable length restrictions.
+    # This is an optional value, it defaults to 120.
+    max_line_length = 120
```

Comment:
In max_line_length also ignored when enable = no?

If not, it could be moved up and the comments for enable can say "the settings below". Or enable can be renamed to enable_colors